ibis/backends/clickhouse/__init__.py

```diff
@@ -197,17 +197,23 @@ def list_databases(self, like: str | None = None) -> list[str]:
         return self._filter_with_like(databases, like)
 
     def list_tables(
-        self, like: str | None = None, database: str | None = None
+        self,
+        like: str | None = None,
+        database: str | None = None,
+        no_views: bool = False,
```

Comment:
generally in this case I prefer the following pattern:
```suggestion
        include_views: bool = True,
```

Reply:
Thanks, makes sense, I will switch to `include_views`.